packages/components/segmented-control/tests/segmented-control.test.tsx

```diff
@@ -58,7 +58,7 @@ describe("<SegmentedControl />", () => {
 
   test("should disable segmented control", () => {
     render(
-      <SegmentedControl data-testid="SegmentedControl" isDisabled>
+      <SegmentedControl data-testid="SegmentedControl" disabled>
```

Comment:
@nathanlao 
We should ensure that the props with `@depricated` are also working, so please do not change the props in the test code.

Reply:
Thank you! I will keep that in mind!